.github/ISSUE_TEMPLATE/test-failure.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Issue Template for Reporting Testing Failures
+about: Describe the failure you encountered while testing
+title: "[Test Case ID]: descriptive title"
+labels: ''
+assignees: ''
+---
+
+**Expected behavior**
+Describe what was expected to happen during testing.
+
+**Actual behavior**
+Describe what actually happened during testing, including any errors or failures encountered.
+
+**Steps to reproduce**
+Provide detailed steps to reproduce the testing failure.
+
+**Release**
+Include information about the release in which the testing was performed.
+
```

Comment:
What information beside release name what do you envision people would have to say. just wondering...

Reply:
I added just the release name for now